### PR TITLE
Duplicate Project (and all illustrations)

### PR DESCRIPTION
**Pull request by aaronalbright**
Ticket:

https://www.notion.so/the-politico/Duplicate-doesn-t-work-b2800c95c85d4d17a5ebe7b146b999af?pvs=4

## Changes
* Explicit CSP permissions for production bundle
* Safer illustration delete with `try/catch` on file and store deletion
* Admin reset button now resets illustrations and settings store

### CODE REVIEW

**Review by brizandrew — CHANGES_REQUESTED** [review]

File: src/components/AdminReset/index.jsx [hunk 1]
```diff
@@ -6,6 +6,7 @@ import store from '../../store';
 export default function AdminReset() {
   const onClick = async () => {
     await store.illustrations.reset();
+    await store.settings.reset();
```
[the remark this answers is not in the document]
Comment (aaronalbright) [reply src/components/AdminReset/index.jsx:9]: I'd assumed "Reset" was meant to wipe the whole thing. That was only way I could get it to let me re-add my AWS keys. I believe at this point that's all resetting the settings does: Wipes your keys, shows you the welcome screen and lets you change your Artisan projects folder. I only put it there so I could do that.

[the remark this answers is not in the document]
Comment (brizandrew) [reply src/components/AdminReset/index.jsx:9]: My plan was to remove this button before launch anyway. This was just for development in my opinion. But if you think we need a reset button on the app, we should consider that?

[the remark this answers is not in the document]
Comment (aaronalbright) [reply src/components/AdminReset/index.jsx:9]: I agree, I'd only use in dev. In a world where we need a user to "start over", which hopefully wouldn't be often, we could direct them how to delete their `com.politico.artisan` folder (after they quit the app, of course) which would accomplish the same thing.

[the remark this answers is not in the document]
Comment (brizandrew) [reply src/components/AdminReset/index.jsx:9]: Ya I agree, okay it's fine then merging as is

**Pull request merged**